.github/workflows/test.yaml

```diff
@@ -46,7 +46,7 @@ jobs:
       - uses: actions/setup-go@d35c59abb061a4a6fb18e82ac0862c26744d6ab5 # v5
         with:
           go-version-file: go.mod
-      - uses: golangci/golangci-lint-action@ec5d18412c0aeab7936cb16880d708ba2a64e1ae # v6.2.0
+      - uses: golangci/golangci-lint-action@4afd733a84b1f43292c63897423277bb7f4313a9 # v8.0.0
```

Comment:
Let's update this in the `validate-action-shas.sh` script as well?

Reply:
Where is that script?  This action seems to be referred here only.